_config.yml

```diff
@@ -68,6 +68,12 @@ defaults:
 twitter:
   username: CrystalLanguage
 
+social:
```

Comment:
What's the point of this? It isn't being used, right?

Reply:
This is configuration for the [seo plugin](https://github.com/jekyll/jekyll-seo-tag)